testdroid_cmdline.sh

```diff
@@ -421,8 +462,55 @@ function get_device_result_file {
 }
 
 
+########################################
+# Get all screenshots for a device
+# Arguments:
+#   test_run_id
+#   device_run_id
+#   device_human_name
+# Returns:
+#   Void (writes files to subfolder TEST_RESULTS_DIR)
+#########################################
+function get_device_screenshots {
+  test_run_id="$1"
+  device_run_id="$2"
+  device_human_name="$3"
+  test_run_item_url=$(url_from_template "${TD_TEST_RUN_ITEM_URL_TEMPLATE}" "${test_run_id}")
+  device_session_screenshots_url="$test_run_item_url/device-runs/$device_session_id/screenshots"
+  response=$(auth_curl "$device_session_screenshots_url")
+  device_screenshot_ids=$(echo "$response" | jq '.data[] |"\(.id);\(.originalName)"')
+  mkdir -p "${TEST_RESULTS_DIR}/screenshots/${device_human_name}"
```

Comment:
you define screenshots path here and then you define the same path inside function. why not to transfer as argument?